packages/react-components/e2e/tests/gauge/gauge.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from '@playwright/test';
+
+const TEST_IFRAME = '#storybook-preview-iframe';
+const TEST_PAGE = '/?path=/story/widgets-gauge-gauge-base--main';
+
+test('guage', async ({ page }) => {
```

Comment:
the tests should also check that every single flag/setting in the `GaugeSettings` type works as expected
this includes: gaugeThickness, showName, showUnit, fontSize, labelFontSize, unitFontSize, yMin, yMax